### ISSUE TITLE
Recommended change to ./settings/language-scl.cson

### ISSUE THREAD

**Issue by ghost** [Gunders89/language-scl#1]
Hello.
I hope this message finds you well.
Thank you for sharing your work with us.. as such I hope to help a little ->

Please find below the change that will make the SCL grammar do indenting and <kbd>Ctrl+/</kbd> will comment out code.

**./settings/language-scl.cson**
```coffee
'.source.scl':
  'editor':
    'autoIndentOnPaste': false
    'autoIndent': true
    'softTabs': true
    'tabLength': 2
    'commentStart': '// '
    'increaseIndentPattern': '^\\s*(if|then|elsif|else|case|[0-9]*\\:|for|while|do|repeat).*$'
    'decreaseIndentPattern': '^\\s*(then|do|end_if;|end_case;|end_for;|end_while;|until|end_repeat;).*$'
    'decreaseNextIndentPattern': '^\\s*(end_if;|end_case;|end_for;|end_while;|until|end_repeat;).*$'
```

Note there is a `.` in-front of `.source.scl`. 

Additionally there is no need for the following directories (and files) to make your grammar work:
`./lib/`
`./menus/`
`./spec/`
`./styles/`
.. but you need to then edit `package.json` to suit.

Please review my suggestion carefully as I might have easily missed something.
(You might consider increasing the version number after this [wink])
**./package.json**
```json
{
  "name": "language-scl",
  "version": "0.2.5",
  "description": "Siemens SCL language support in Atom",
  "keywords": [],
  "homepage": "https://github.com/Gunders89/language-scl#readme",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/Gunders89/language-scl.git"
  },
  "license": "MIT",
  "engines": {
    "atom": ">=1.0.0 <2.0.0"
  },
  "dependencies": {},
  "devDependencies": {},
  "readme": "# language-scl package for Atom\n\nAdds syntax highlighting and snippets to SCL files in Atom.\n\nContributions are greatly appreciated. Please fork this repository and open a pull request to add snippets, make grammar changes, etc.\n",
  "readmeFilename": "README.md"
}
```

Best regards.
\- Dan Padric

**Comment by Gunders89** [comment]
Thank you for your input :)

I'm on it.

**Comment by Gunders89** [comment]
Part of the fix can be found in f381ef9

**Comment by Gunders89** [comment]
Part of the fix can be found in d16a6a8

**Comment by ghost** [comment]
_**Nice!**_ Thank you.
I see I have missed 'keymaps' - sorry, this now points at a function that doe not exist.
A basic grammar package does not need 'keymaps' either.
Well wishes.
